### PR TITLE
protobuf: 3.12.4

### PR DESCRIPTION
**Pull request by AustinShalit**
Supersedes #58104

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/protobuf.rb [hunk 1]
```diff
@@ -1,25 +1,21 @@
 class Protobuf < Formula
   desc "Protocol buffers (Google's data interchange format)"
   homepage "https://github.com/protocolbuffers/protobuf/"
-  url "https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protobuf-all-3.12.3.tar.gz"
-  sha256 "1a83f0525e5c8096b7b812181865da3c8637de88f9777056cefbf51a1eb0b83f"
-  revision 1
+  url "https://github.com/protocolbuffers/protobuf.git",
+      tag:      "v3.12.4",
+      revision: "c9d2bd2fc781fe67ebf306807b9b6edb4a0d2764"
```
Comment (SeekingMeaning) [inline Formula/protobuf.rb:6]:
```suggestion
  url "https://github.com/protocolbuffers/protobuf/releases/download/v3.12.4/protobuf-all-3.12.4.tar.gz"
  sha256 "e7bf41873d1a87c05c2b0a6197f4445c6ea3469ce0165ff14de2df8b34262530"
```

My fault, I misinterpreted your question in #58104

**Review by SeekingMeaning — DISMISSED** [review]

File: Formula/protobuf.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
 class Protobuf < Formula
   desc "Protocol buffers (Google's data interchange format)"
   homepage "https://github.com/protocolbuffers/protobuf/"
-  url "https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protobuf-all-3.12.3.tar.gz"
-  sha256 "1a83f0525e5c8096b7b812181865da3c8637de88f9777056cefbf51a1eb0b83f"
-  revision 1
```
Comment (AustinShalit) [inline Formula/protobuf.rb:6]: Should I have removed this?

Comment (SeekingMeaning) [reply Formula/protobuf.rb:6]: Yes

Comment (AustinShalit) [reply Formula/protobuf.rb:6]: Cool

**Pull request closed without merging**